docs/kubernetes/backup.md

```diff
@@ -0,0 +1,174 @@
+                                }
+                            },
+                            "manager": "kubectl-create",
+                            "operation": "Update",
+                            "time": "2020-10-30T06:47:29Z"
+                        },
+                        {
+                            "apiVersion": "snapshot.storage.k8s.io/v1beta1",
+                            "fieldsType": "FieldsV1",
+                            "fieldsV1": {
+                                "f:metadata": {
+                                    "f:finalizers": {
+                                        ".": {},
+                                        "v:\"snapshot.storage.kubernetes.io/volumesnapshot-as-source-protection\"": {},
+                                        "v:\"snapshot.storage.kubernetes.io/volumesnapshot-bound-protection\"": {}
+                                    }
+                                },
+                                "f:status": {
+                                    ".": {},
+                                    "f:boundVolumeSnapshotContentName": {},
+                                    "f:creationTime": {},
+                                    "f:readyToUse": {},
+                                    "f:restoreSize": {}
+                                }
+                            },
+                            "manager": "snapshot-controller",
+                            "operation": "Update",
+                            "time": "2020-10-30T06:48:37Z"
+                        }
+                    ],
+                    "name": "backup-source-pvc",
+                    "namespace": "default",
+                    "resourceVersion": "1484",
+                    "uid": "38d149bc-1c26-41d6-9c19-de97950997ee"
+                },
+                "spec": {
+                    "source": {
+                        "persistentVolumeClaimName": "source-pvc"
+                    },
+                    "volumeSnapshotClassName": "csi-gcp-filestore-backup-snap-class"
+                },
+                "status": {
+                    "boundVolumeSnapshotContentName": "snapcontent-38d149bc-1c26-41d6-9c19-de97950997ee",
+                    "creationTime": "2020-10-30T06:48:16Z",
+                    "readyToUse": true,
+                    "restoreSize": "1Ti"
+                }
+            }
+        ],
+        "kind": "List",
+        "metadata": {
+            "resourceVersion": "",
+            "selfLink": ""
+        }
+    }
+    ```
+
+7. Restore the `VolumeSnapshot` into a new PVC:
+
+    Create a new PVC. Specify `spec.dataSource` section to restore from VolumeSnapshot `backup-source-pvc`.
+
+    ```console
+    kubectl create -f ./examples/kubernetes/backups-restore/restored-pod-pvc.yaml
+    ```
+
+8. Verify sample data has been restored:
+
+    Create a sample pod with the restored PVC:
+
+    ```console
+    kubectl create -f ./examples/kubernetes/snapshot/restored-pod.yaml
```

Comment:
Got a "does not exist" error.

Reply:
thanks for catching this. changed the yaml midway writing the readme.